src/community/security/oauth2-openid-connect/oauth2-openid-connect-web/src/main/resources/GeoServerApplication.properties

```diff
@@ -89,6 +89,11 @@ OpenIdConnectAuthProviderPanel.sendClientSecretHelp.title=Send Client Secret in
 OpenIdConnectAuthProviderPanel.sendClientSecretHelp= Check this option if the OpenId server requires a confidential client \
   to send the client_secret in the token response (eg. ADFS and Azure AD).
 
+OpenIdConnectAuthProviderPanel.allowUnSecureLogging=Log Sensitive Information (do not use in production)
+OpenIdConnectAuthProviderPanel.allowUnSecureLoggingHelp.title=Log Sensitive Information
+OpenIdConnectAuthProviderPanel.allowUnSecureLoggingHelp=Check this option if you want to log more sensitive information \
+  (i.e. tokens).  This is useful when debugging an OIDC configuration that isn't working correctly. 
```

Comment:
Appreciate the direction because the user manual is too far away.

You still may wish to update the user manual page as part of this pull request.

Reply:
I will wait for @taba90